samples/sensor/stm32_vbat_sensor/boards/nucleo_g071rb.overlay

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2022 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Application overlay for creating vbat sensor device instance
+ */
+
+/ {
+	stm-vbat {
+		compatible = "st,stm32-vbat";
+		ratio = <3>;
+		io-channels = <&adc1 14>;
+		status = "okay";
+		label = "VBAT";
+	};
+};
+
+&adc1 {
+    vref-mv = <3000>;
```

Comment:
Nits: Use tab for indentation and not spaces.

Reply:
done